content/app/development/ux/images/_index.en.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Images
+description: How to add and edit image references.
+toc: true
+weight: 40
+---
+
+## Add images to the application
+
+Adding images is dont in _FormLayout.json_ by using the image component. Alternative text for the image can be added as a text resource, defined in _resource.[språk].json_.
```

Comment:
Dont?

Reply:
Whops, this should of course be 'done'. Ill fix!

- [ ] fix typo

Reply:
```suggestion
Adding images is don't in _FormLayout.json_ by using the image component. Alternative text for the image can be added as a text resource, defined in _resource.[språk].json_.
```